### ISSUE TITLE
Latest schema update breaks `pypa` source.

### ISSUE THREAD

**Issue by Kurt-von-Laven** [twu/skjold#68]
Excerpt from pyproject.toml:

```toml
  [tool.skjold]
  sources = ["pypa"]
```

Excerpt from pre-commit output when running `git commit`:

```
skjold........................................................................Failed
- hook id: skjold
- exit code: 1

Traceback (most recent call last):
  File "~/.cache/pre-commit/repo0vl87024/py_env-python3/bin/skjold", line 8, in <module>
    sys.exit(cli())
  File "~/.cache/pre-commit/repo0vl87024/py_env-python3/lib/python3.9/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "~/.cache/pre-commit/repo0vl87024/py_env-python3/lib/python3.9/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "~/.cache/pre-commit/repo0vl87024/py_env-python3/lib/python3.9/site-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "~/.cache/pre-commit/repo0vl87024/py_env-python3/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "~/.cache/pre-commit/repo0vl87024/py_env-python3/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "~/.cache/pre-commit/repo0vl87024/py_env-python3/lib/python3.9/site-packages/click/decorators.py", line 84, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "~/.cache/pre-commit/repo0vl87024/py_env-python3/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "~/.cache/pre-commit/repo0vl87024/py_env-python3/lib/python3.9/site-packages/skjold/cli.py", line 184, in audit_
    findings = audit(config, packages, ignore=ignore)
  File "~/.cache/pre-commit/repo0vl87024/py_env-python3/lib/python3.9/site-packages/skjold/tasks.py", line 221, in audit
    if source.has_security_advisory_for(package_name):
  File "~/.cache/pre-commit/repo0vl87024/py_env-python3/lib/python3.9/site-packages/skjold/sources/pypa.py", line 59, in has_security_advisory_for
    return package_name.strip().lower() in self.advisories.keys()
  File "~/.cache/pre-commit/repo0vl87024/py_env-python3/lib/python3.9/site-packages/skjold/models.py", line 104, in advisories
    self.populate_from_cache()
  File "~/.cache/pre-commit/repo0vl87024/py_env-python3/lib/python3.9/site-packages/skjold/sources/pypa.py", line 40, in populate_from_cache
    self._advisories[advisory.package_name.lower()].append(advisory)
  File "~/.cache/pre-commit/repo0vl87024/py_env-python3/lib/python3.9/site-packages/skjold/sources/osv.py", line 80, in package_name
    return str(self._json["package"]["name"]).strip()
KeyError: 'package'
```

**Comment by twu** [comment]
Yes, sorry about that. They _recently_ introduced breaking changes into their schema. Should be fixed with #67.

**Comment by Kurt-von-Laven** [comment]
Thank you for the speedy fix! I thought I had lost my mind when I looked at the other ticket again until I realized that it originally said `pyup` ha ha ha.
